### PR TITLE
create quote from back-office

### PR DESCRIPTION
**Pull request by AlexSJ1990**

### CODE REVIEW

File: src/main/kotlin/com/hedvig/underwriter/serviceIntegration/productPricing/ProductPricingClient.kt [hunk 1]
```diff
@@ -79,4 +82,9 @@ interface ProductPricingClient {
         @Valid @RequestBody request: CalculateBundleInsuranceCostRequest,
         @PathVariable memberId: String
     ): ResponseEntity<InsuranceCost>
+
+    @GetMapping("/_/agreements/{agreementId}")
+    fun getAgreementFromAgreementId(
```
Comment (FredrikAreschoug) [inline src/main/kotlin/com/hedvig/underwriter/serviceIntegration/productPricing/ProductPricingClient.kt:87]: I think just `getAgreement` is fine

File: src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt [hunk 2]
```diff
@@ -148,6 +148,36 @@ class QuoteServiceImpl(
         return transformCompleteQuoteReturn(breachedGuidelinesOrQuote, quoteId)
     }
 
+    override fun createQuoteFromBackOffice(
```
Comment (FredrikAreschoug) [inline src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt:151]: In my opinion this should be `createQuoteFromAgreement`

File: src/main/kotlin/com/hedvig/underwriter/web/QuoteController.kt [hunk 3]
```diff
@@ -71,6 +72,20 @@ class QuoteController @Autowired constructor(
             ).getOrHandle { it }
     }
 
+    @PostMapping("/createQuoteFromBackOffice")
```
Comment (FredrikAreschoug) [inline src/main/kotlin/com/hedvig/underwriter/web/QuoteController.kt:75]: If you like the naming of from agreement change the name of this endpoint and the dto as well :)

File: src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt (hunk 2 above)
Comment (vonElfvin) [reply src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt]: I agree

**Pull request merged**